### PR TITLE
Bug 1588590 - fix GCP mobile worker configs for prod/dev

### PR DESCRIPTION
**Pull request by MihaiTabara**
This is to match the `script_config.json` we generate on puppet [here](https://github.com/mozilla-releng/build-puppet/blob/master/modules/pushapk_scriptworker/manifests/settings.pp)

### CODE REVIEW

File: pushapkscript/docker.d/worker.yml [hunk 1]
```diff
@@ -97,21 +102,34 @@ products:
...
+          skip_checks_fennec: true
+          skip_check_signature: true
+          skip_check_ordered_version_codes: true
+          override_channel_model: "single_google_app"
+          apps:
+            production:
+              package_names:
+                - "org.mozilla.tv.firefox"
+              amazon:
+                client_id: 'dummy'
```
Comment (MihaiTabara) [inline pushapkscript/docker.d/worker.yml:122]: I'll amend later this week these to reflect the correct env var once I sent it out to Jeremy and update cloudops-infra repo too.

Comment (MihaiTabara) [reply pushapkscript/docker.d/worker.yml:122]: Just to clarify - we're not deploying this to `firefox-tv` repo until we add these missing credentials.

File: pushapkscript/docker.d/worker.yml [hunk 2]
```diff
@@ -97,21 +102,34 @@ products:
...
             certificate_alias: "reference-browser"
-
+        - product_names: [ "firefox-tv" ]
+          skip_check_multiple_locales: true
+          skip_check_same_locales: true
+          skip_checks_fennec: true
+          skip_check_signature: true
+          skip_check_ordered_version_codes: true
+          override_channel_model: "single_google_app"
+          apps:
```
Comment (mitchhentges) [inline pushapkscript/docker.d/worker.yml:117]: If the channel model for the dep `firefox-tv` is `single_google_app`, then this property should change from `apps` to `app` and it should be more google-specific, like [this one](https://github.com/mozilla-releng/scriptworker-scripts/pull/51/files#diff-3ab502ad0f8a21fc4b9a368b6cbcbd8eR105)

**Review by mitchhentges — CHANGES_REQUESTED** [review]

File: pushapkscript/docker.d/worker.yml (hunk 2 above)
Comment (MihaiTabara) [reply pushapkscript/docker.d/worker.yml:117]: Hm, dep `firefox-tv` has no `single_google_app` channel mode.

Comment (MihaiTabara) [reply pushapkscript/docker.d/worker.yml:117]: I copy-pasted from puppet, hm.

Comment (mitchhentges) [reply pushapkscript/docker.d/worker.yml:117]: I think that puppet is wrong. You can confirm this by looking at [how `pushapkscript` parses the publish config](https://github.com/mozilla-releng/scriptworker-scripts/blob/master/pushapkscript/pushapkscript/publish_config.py#L21) (note how it looks for an `app` property, not `apps`)

**Review by mitchhentges — APPROVED** [review]
If this is the same as `puppet`, feel free to land. I'll update both GCP and puppet

**Review by mitchhentges — CHANGES_REQUESTED** [review]
Are you sure you grabbed that from the up-to-date puppet configs?
Looking at the `mobile-dep` config for `firefox-tv`, its channel model [isn't `single_google_app`](https://github.com/mozilla-releng/build-puppet/blob/master/modules/pushapk_scriptworker/manifests/settings.pp#L200). The puppet config should be correct

File: pushapkscript/docker.d/worker.yml (hunk 2 above)
Comment (mitchhentges) [reply pushapkscript/docker.d/worker.yml:117]: I misunderstood, the dep `firefox-tv` config hasn't had `override_channel_model: "single_google_app"`, and should continue to not have that model. For some reason, I assumed that dep `firefox-tv` was like dep `reference-browser` and dep `fenix` in having that single dummy app.

Remove line 117 and this should be good to land!

**Review by mitchhentges — APPROVED** [review]
Mastapeece

**Pull request merged**